Automattic/woocommerce-payments
# refactor: replace survey notices with Gutenberg

Fixes https://github.com/Automattic/woocommerce-payments/issues/2545

#### Changes proposed in this Pull Request









- Added a blue background to the info notices
- Replaced the notices in the opt-out surveys with the `Notice` component from Gutenberg

![Screen Shot 2021-07-23 at 4 18 01 PM](https://user-images.githubusercontent.com/273592/126842863-776478b1-0b8f-4254-ac7a-6f5f4e9b918e.png)
![Screen Shot 2021-07-23 at 4 17 20 PM](https://user-images.githubusercontent.com/273592/126842864-33b9f962-4be1-449f-95ce-9aeab6624996.png)
![Screen Shot 2021-07-23 at 4 16 12 PM](https://user-images.githubusercontent.com/273592/126842865-a164d941-5970-492c-90de-e98058d77071.png)


#### Testing instructions





- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Enable/Disable UPE in order to interact with the opt-out survey and see the new notices

-------------------
